### PR TITLE
Add some drug ordering metadata

### PR DESCRIPTION
**Pull request by rbuisson**
Please enter the commit message for your changes. Lines starting

### CODE REVIEW

**Comment by rbuisson** [comment]
Still a draft PR at this stage since we I encounter 2 or 3 errors in Iniz related to some dosage form concepts already existing.
"Tablet", "Spray", "Suppository" I believe.

**Comment by rbuisson** [comment]
Now missing to fix is the order frequencies.
We shall get them from OCL instead of a CSV. Looking into it tmr.

**Comment by rbuisson** [comment]
Should be good to go.

File: distro/configuration/appointmentservicedefinitions/servicedefinitions.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Uuid,Void/Retire,Name,Description,Duration,Start Time,End Time,Max Load,Speciality,Location,Label Colour
 be6c3bea-f7d6-4686-9de3-ee799d68a63f,,Malnutrition,,,,,,14106bb7-dad6-4446-809d-737f4c128ae3,,#FFFF00
-09c85fcf-a43e-424a-b3e3-8671b09eda5d,,General,,,,,,9f2a8cd0-32c6-4844-8df7-1ac9c4d79943,,#b533ff
+09c85fcf-a43e-424a-b3e3-8671b09eda5d,,General,,,,,,9f2a8cd0-32c6-4844-8df7-1ac9c4d79943,,#b533ff
```
Comment (ibacher) [inline distro/configuration/appointmentservicedefinitions/servicedefinitions.csv:3]:
```suggestion
09c85fcf-a43e-424a-b3e3-8671b09eda5d,,General,,,,,,9f2a8cd0-32c6-4844-8df7-1ac9c4d79943,,#b533ff

```

File: distro/configuration/appointmentservicetypes/serviceTypes.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
 Void/Retire,UUID,Name,Service definition,Duration,_order:1000
-,fbec4378-2d0d-4509-a56e-be0a53700709,Short follow-up,General,10,
+,fbec4378-2d0d-4509-a56e-be0a53700709,Short follow-up,General,10,
```
Comment (ibacher) [inline distro/configuration/appointmentservicetypes/serviceTypes.csv:2]:
```suggestion
,fbec4378-2d0d-4509-a56e-be0a53700709,Short follow-up,General,10,

```

File: distro/configuration/appointmentspecialities/specialities.csv [hunk 3]
```diff
@@ -1,4 +1,3 @@
 Uuid,Void/Retire,Name
 9f2a8cd0-32c6-4844-8df7-1ac9c4d79943,,General
-14106bb7-dad6-4446-809d-737f4c128ae3,,Pediatrics
-
+14106bb7-dad6-4446-809d-737f4c128ae3,,Pediatrics
```
Comment (ibacher) [inline distro/configuration/appointmentspecialities/specialities.csv:3]:
```suggestion
14106bb7-dad6-4446-809d-737f4c128ae3,,Pediatrics

```

File: distro/configuration/concepts/drugs_concepts.csv [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+05130695-8005-4d57-a803-111265f0f4c1,,,Diclofenac & Vitamin B complex,Diclofénac & Complexe de vitamine B,Drug,N/A,,,
+66c997ab-d252-43e5-94ca-b5925a60fdbb,,,Diclofenac & Methocarbamol & Paracetamol,Diclofenac & Methocarbamol & Paracetamol,Drug,N/A,,,
+5ac2861d-71e9-4488-bf29-449d555ddfcb,,,Diosmine & Hesperidine,Diosmine & Hesperidine,Drug,N/A,,,
+909ea570-031b-422e-b28d-17b7f492a1b9,,,Gemfibrozil,Genfibrozil,Drug,N/A,,,
+1d47e474-4ae6-4e47-9b69-2ce6ed7399ac,,,Glucosamine & Chondroitin,Glucosamine & Chondroitin,Drug,N/A,,,
+a30dc1f2-a26b-4bc0-90a4-e7eb7feea473,,,Nystatin & Zinc Oxide,Nystatine et oxyde de zinc,Drug,N/A,,,
+bcf73538-3f39-4429-a30d-8499609942cd,,,Multivitamin & Iron,Multivitamines et fer,Drug,N/A,,,
+d644e9bd-7ae2-4ef3-b7b7-739816051213,,,Pentoxifylline,Pentoxifylline,Drug,N/A,,,
+792871a0-4a52-4e5d-bdd2-4fd49d636227,,,Podophillin,Podophylline,Drug,N/A,,,
+916e4268-45b9-4db4-b273-1d5b4a6539b3,,,Bismuth subsalicylate,Sous-Salicylate de bismuth,Drug,N/A,,,
```
Comment (ibacher) [inline distro/configuration/concepts/drugs_concepts.csv:488]:
```suggestion
916e4268-45b9-4db4-b273-1d5b4a6539b3,,,Bismuth subsalicylate,Sous-Salicylate de bismuth,Drug,N/A,,,

```

File: distro/configuration/drugs/drugs.csv [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+af3fce1f-dcb1-4f76-ad4c-b8ebda43070c,,Vitamine B-Complex Injection,836b7f4c-9e3c-4a53-bb08-ab8990b5b9d8,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,N/A,
+481e357d-6c45-4c8a-beec-6d8f858f2552,,Vitamine B-Complex Co,836b7f4c-9e3c-4a53-bb08-ab8990b5b9d8,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,N/A,
+8a54b957-8c4d-4ab6-9e42-575bf5b7be83,,Vitamin A,54528cf8-0022-4f32-a038-cbb2d5965ae5,90a3f930-2eef-4a2e-8427-5359607c7b13,N/A,
+9eb0a661-f27b-472e-a35d-04dc1dba0860,,Mycrogynon 30,b2727136-70f5-470b-81a2-26aeff3f346f,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,N/A,
+61444a80-77b6-4986-b9c1-07eab731ce28,,Microlut 30,71eb922d-5a51-415a-a838-8a815280abfa,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,N/A,
+012f0f5d-c76e-4a28-8e60-30c440f5ff62,,Depo Provera,6533b40f-a783-435f-9c92-19d5e7343d98,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,N/A,
+62a34f17-e83c-4dbc-b53b-9a7f9478501a,,RHEZ,c103ec05-577d-4f64-85ec-54c361895198,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,N/A,
+667682cc-6731-4784-acdc-c7fb7c98d15a,,RH,606f9339-1430-4a00-8515-55095ef66c3b,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,N/A,
```
Comment (ibacher) [inline distro/configuration/drugs/drugs.csv:321]:
```suggestion
667682cc-6731-4784-acdc-c7fb7c98d15a,,RH,606f9339-1430-4a00-8515-55095ef66c3b,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,N/A,

```

File: distro/configuration/orderfrequencies/orderfrequencies.csv [hunk 6]
```diff
@@ -7,5 +7,4 @@ af7c1fe6-d669-414e-b066-e9733f0de7a8,,2,Every two hours,,
 136ebdb1-e984-47cf-8ec2-4e6b2ffe0ab8,,6,Every four hours,,
 336ebdb5-e985-47cf-8mc2-4e8b2ffe0a83,,8,Every three hours,,
 736ebdb7-9989-47cf-8ec2-4e8b2ffe0ab2,,12,Every twelve hours,,
-636ebdb4-e988-37cf-8ec2-4e9b2ffe8ab3,,24,Every hour,,
-
+636ebdb4-e988-37cf-8ec2-4e9b2ffe8ab3,,24,Every hour,,
```
Comment (ibacher) [inline distro/configuration/orderfrequencies/orderfrequencies.csv:10]:
```suggestion
636ebdb4-e988-37cf-8ec2-4e9b2ffe8ab3,,24,Every hour,,

```

File: distro/configuration/patientidentifiertypes/patientidentifiertypes.csv [hunk 7]
```diff
@@ -2,4 +2,4 @@ Uuid,Void/Retire,Name,Description,Required,Format,Format description,Validator,L
 05a29f94-c0ed-11e2-94be-8c13b969e334,,OpenMRS ID,"OpenMRS patient identifier, with check-digit",TRUE,,,org.openmrs.module.idgen.validator.LuhnMod30IdentifierValidator,NOT_USED,UNIQUE
 22348099-3873-459e-a32e-d93b17eda533,,Legacy ID,Identifier type to record optional previous identifers ,FALSE,,,,NOT_USED,
 b4143563-16cd-4439-b288-f83d61670fc8,,ID Card,ID Card,FALSE,,,,NOT_USED,UNIQUE
-a71403f3-8584-4289-ab41-2b4e5570bd45,,SSN,Social Security Number,FALSE,[A-Z]{1}-[0-9]{7},"Identifier should be 1 letter, followed by a dash and 7 numerical characters. Eg, A-0010902",,NOT_USED,UNIQUE
+a71403f3-8584-4289-ab41-2b4e5570bd45,,SSN,Social Security Number,FALSE,[A-Z]{1}-[0-9]{7},"Identifier should be 1 letter, followed by a dash and 7 numerical characters. Eg, A-0010902",,NOT_USED,UNIQUE
```
Comment (ibacher) [inline distro/configuration/patientidentifiertypes/patientidentifiertypes.csv:5]:
```suggestion
a71403f3-8584-4289-ab41-2b4e5570bd45,,SSN,Social Security Number,FALSE,[A-Z]{1}-[0-9]{7},"Identifier should be 1 letter, followed by a dash and 7 numerical characters. Eg, A-0010902",,NOT_USED,UNIQUE

```

**Review by ibacher — APPROVED** [review]
LGTM!

**Pull request merged**